src/Domain/DomainCreators/Sphere.cpp

```diff
@@ -0,0 +1,112 @@
+      with_equiangular_map_(std::move(with_equiangular_map)) {}  // NOLINT
+
+Domain<3, Frame::Inertial> Sphere::create_domain() const noexcept {
+  using Wedge3DMap = CoordinateMaps::Wedge3D;
+  using Wedge3DType =
+      CoordinateMap<Frame::Logical, Frame::Inertial, Wedge3DMap>;
+  using AffineMap = CoordinateMaps::AffineMap;
+  using AffineMap3D =
+      CoordinateMaps::ProductOf3Maps<AffineMap, AffineMap, AffineMap>;
+
+  std::vector<std::array<size_t, 8>> corners{
+      {{7, 5, 8, 6, 15, 13, 16, 14}},  // Upper z
+      {{4, 2, 3, 1, 12, 10, 11, 9}},   // Lower z
+      {{4, 8, 2, 6, 12, 16, 10, 14}},  // Upper y
+      {{7, 3, 5, 1, 15, 11, 13, 9}},   // Lower y
+      {{1, 2, 5, 6, 9, 10, 13, 14}},   // Upper x
+      {{4, 3, 8, 7, 12, 11, 16, 15}},  // Lower x
+      {{3, 1, 4, 2, 7, 5, 8, 6}}};     // center cube
+
+  return Domain<3, Frame::Inertial>{
+      make_vector<std::unique_ptr<
+          CoordinateMapBase<Frame::Logical, Frame::Inertial, 3>>>(
+          std::make_unique<Wedge3DType>(Wedge3DMap(inner_radius_, outer_radius_,
+                                                   Direction<3>::upper_zeta(),
+                                                   0, with_equiangular_map_)),
+          std::make_unique<Wedge3DType>(Wedge3DMap(inner_radius_, outer_radius_,
+                                                   Direction<3>::lower_zeta(),
+                                                   0, with_equiangular_map_)),
+          std::make_unique<Wedge3DType>(Wedge3DMap(inner_radius_, outer_radius_,
+                                                   Direction<3>::upper_eta(), 0,
+                                                   with_equiangular_map_)),
+          std::make_unique<Wedge3DType>(Wedge3DMap(inner_radius_, outer_radius_,
+                                                   Direction<3>::lower_eta(), 0,
+                                                   with_equiangular_map_)),
+          std::make_unique<Wedge3DType>(Wedge3DMap(inner_radius_, outer_radius_,
+                                                   Direction<3>::upper_xi(), 0,
+                                                   with_equiangular_map_)),
+          std::make_unique<Wedge3DType>(Wedge3DMap(inner_radius_, outer_radius_,
+                                                   Direction<3>::lower_xi(), 0,
+                                                   with_equiangular_map_)),
+          std::make_unique<
+              CoordinateMap<Frame::Logical, Frame::Inertial, AffineMap3D>>(
+              AffineMap3D{AffineMap(-1.0, 1.0, -1.0 * inner_radius_ / sqrt(3.0),
+                                    inner_radius_ / sqrt(3.0)),
+                          AffineMap(-1.0, 1.0, -1.0 * inner_radius_ / sqrt(3.0),
+                                    inner_radius_ / sqrt(3.0)),
+                          AffineMap(-1.0, 1.0, -1.0 * inner_radius_ / sqrt(3.0),
+                                    inner_radius_ / sqrt(3.0))})),
```

Comment:
If I correctly understand this, the central cube does not (yet) have an `equiangular_map` option, which means the grid will not be conforming (i.e. the grid points will not line up) at the cube-to-wedge interfaces when `with_equiangular_map_ == true`.

Do we want to generalize the cube in this PR or make an issue to remind us to come back to this?

Reply:
Thanks for catching this :smiley:  I think the equiangular cube will go in its own PR